SUI/SUI/Main/MainView.swift

```diff
@@ -0,0 +1,137 @@
+        static let startButtonTitle = "GET STARTED"
+        static let questionString = "Don't have an account?"
+        static let navigationTitleString = "Sing in here"
+        static let questionImageName = "questionmark"
+        static let darkPurpleColorName = "darkPurple"
+        static let opacityNumber = 0.9
+        static let spaceNumber: CGFloat = 60
+        static let tenSpaceNumber: CGFloat = 10
+    }
+
+    // MARK: - Public Properties
+
+    var body: some View {
+        NavigationView {
+            ZStack {
+                gradientView
+                VStack {
+                    Spacer(minLength: Constants.spaceNumber)
+                    titleTextView
+                    asyncImageView
+                    Spacer()
+                    startButtonView
+                    navigationLinkView
+                    VStack(spacing: Constants.tenSpaceNumber) {
+                        questionTextView
+                        loginNavigationLinkView
+                            .padding(.bottom, Constants.spaceNumber)
+                    }
+                }
+            }
+            .ignoresSafeArea()
+        }
+        .onTapGesture {
+            hideKeyboard()
+        }
+    }
+
+    // MARK: - Private Properties
+
+    @StateObject private var mainViewModel = MainViewModel()
+
+    private var titleTextView: some View {
+        Text(Constants.titleString)
+            .foregroundColor(.white)
+            .font(.system(.largeTitle, design: .default))
+            .bold()
+            .multilineTextAlignment(.center)
+    }
+
+    private var startButtonView: some View {
+        Button(Constants.startButtonTitle) {
+            mainViewModel.isShowChair = true
+        }
+        .frame(width: 320, height: 80)
+        .background(.white)
+        .foregroundColor(Color(Constants.darkPurpleColorName))
+        .font(.system(size: 28, weight: .bold, design: .default))
+        .cornerRadius(40)
+        .padding(30)
+        .padding(.bottom, 20)
+    }
+
+    private var navigationLinkView: some View {
+        NavigationLink(isActive: $mainViewModel.isShowChair) {
+            ChairView()
+        } label: {
+            EmptyView()
+        }
+    }
+
+    private var questionTextView: some View {
+        Text(Constants.questionString)
+            .foregroundColor(.white)
+            .font(.system(size: 20, weight: .bold, design: .rounded))
+    }
+
+    private var loginNavigationLinkView: some View {
+        NavigationLink(isActive: $mainViewModel.isShowLogin) {
+            LoginView()
+        } label: {
+            Text(Constants.navigationTitleString)
+                .foregroundColor(.white)
+                .font(.system(size: 35, weight: .bold, design: .default))
+                .underline(true, color: .white)
+        }
+    }
+
+    private var asyncImageView: some View {
+        AsyncImage(
+            url: URL(string: mainViewModel.urls[Int.random(in: 0 ... 6)])
```

Comment:
```suggestion
            url: URL(string: mainViewModel.urls[Int.random(in: 0 ... 6)])
```
вообще вроде все нужно вынести во вьюмодель. но у меня не получислось
вынеси хотя бы генерацию урлов во вьюмодель